### PR TITLE
[13.0][IMP] account_invoice_export custom url

### PR DESCRIPTION
**Pull request by TDu**
Add the option to customize the url used to export dynamically.

### CODE REVIEW

**Review by simahawk — APPROVED** [review]

**Comment by TDu** [comment]
I added one fix if url is empty.
And the same for user and password

File: account_invoice_export/models/transmit_method.py [hunk 1]
```diff
@@ -21,6 +21,14 @@ def get_transmission_http_header(self):
 
         """
         self.ensure_one()
-        auth = "{}:{}".format(self.destination_user, self.destination_pwd,)
+        auth = "{}:{}".format(self.destination_user or "", self.destination_pwd or "")
```
Comment (simahawk) [inline account_invoice_export/models/transmit_method.py:26]: wouldn't be better to raise a specific error?

Comment (TDu) [reply account_invoice_export/models/transmit_method.py:26]: Well my thought was letting the call be made and let `request` or the distant server raise an error.
From what I know the password could be empty ?

Comment (simahawk) [reply account_invoice_export/models/transmit_method.py:26]: yeah but the error might be not so clear for the user. I guess destination user is mandatory? If yes, raising a UserError here or before should be better. Unless I miss something else.

Comment (TDu) [reply account_invoice_export/models/transmit_method.py:26]: From what  I read online and what people understand from the  specs https://datatracker.ietf.org/doc/html/rfc1945#section-11 Basic Authentication allows for an empty user name and password. That is sending just the semi column is a valid option.
In that light the code is correct.

**Comment by TDu** [comment]
@simahawk  I answered our discussion above and rebased the pr

**Pull request merged**